ndk/src/aaudio.rs

```diff
@@ -229,7 +229,7 @@ impl AAudioError {
 
 pub type Result<T, E = AAudioError> = std::result::Result<T, E>;
 
-fn construct<T>(with_ptr: impl FnOnce(*mut T) -> ffi::camera_status_t) -> Result<T> {
+fn construct<T>(with_ptr: impl FnOnce(*mut T) -> ffi::aaudio_result_t) -> Result<T> {
```

Comment:
What's this about?

Reply:
It doesn't do much since both are just typedefs for `c_int`/`i32`, but the `aaudio` one should be used in this case because it handles aaudio functions.

Reply:
Gotcha, thanks for catching this!